probe-rs/src/debug/variable.rs

```diff
@@ -549,7 +543,7 @@ impl Variable {
     }
 
     /// Implementing get_value(), because Variable.value has to be private (a requirement of updating the value without overriding earlier values ... see set_value()).
-    pub fn get_value(&self, variable_cache: &variable_cache::VariableCache) -> String {
+    pub fn get_value(&self, variable_cache: &VariableCache) -> String {
```

Comment:
At the same time, can we move away from `get_` prefixes? We should prefer the Rust API guideline recommendations (to_, into_, as_). Also, this fn returns a value **representation**, not the value itself.

Reply:
Yeah, I meant to not touch the general API, but lets do it :)